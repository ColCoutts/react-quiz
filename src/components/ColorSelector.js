import React from 'react';
import PropTypes from 'prop-types';

function ColorSelector({ color, backgroundColor, handleChange }) {
  
  return (
    <>
      <input name="color" type="color" value={color} onChange={handleChange}></input>
      <input name="backgroundColor" type="color" value={backgroundColor} onChange={handleChange}></input>
    </>
  );
}

ColorSelector.propTypes = {
  color: PropTypes.string.isRequired,
  backgroundColor: PropTypes.string.isRequired,
  handleChange: PropTypes.func.isRequired
};

export default ColorSelector;
