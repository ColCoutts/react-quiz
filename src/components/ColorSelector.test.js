import React from 'react';
import { shallow } from 'enzyme';
import ColorSelector from './ColorSelector';

describe('textSelector component', () => {
  it('renders a snapshot', () => {
    const color = 'blue';
    const backgroundColor = 'green';
    const wrapper = shallow(<ColorSelector color={color} backgroundColor={backgroundColor} />);
    expect(wrapper).toMatchSnapshot();
  });
});
