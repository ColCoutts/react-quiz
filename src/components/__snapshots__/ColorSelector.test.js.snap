// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`textSelector component renders a snapshot 1`] = `
<Fragment>
  <input
    name="color"
    type="color"
    value="blue"
  />
  <input
    name="color"
    type="color"
    value="green"
  />
</Fragment>
`;
